src/Microsoft.DotNet.Build.Tasks.Packaging/src/GenerateNuSpec.cs

```diff
@@ -197,7 +197,7 @@ private List<ManifestFile> GetManifestFiles()
                     select new ManifestFile()
                     {
                         Source = f.GetMetadata(Metadata.FileSource),
-                        Target = f.GetMetadata(Metadata.FileTarget),
+                        Target = f.GetMetadata(Metadata.FileTarget).Replace('/', Path.DirectorySeparatorChar),
```

Comment:
Sorry for the confusion, I just didn't know this:

>  We generate the target path

I agree that `\` and `/` have the same problem, I was just worried that someone might break the convention and end up with mixed slashes. Thanks for the extra info.